<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddImageDialog</class>
 <widget class="QDialog" name="AddImageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>143</red>
        <green>86</green>
        <blue>59</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>181</green>
        <blue>64</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>181</green>
        <blue>64</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>243</green>
        <blue>225</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>143</red>
        <green>86</green>
        <blue>59</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>181</green>
        <blue>64</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>181</green>
        <blue>64</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>243</green>
        <blue>225</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>120</red>
        <green>120</green>
        <blue>120</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>181</green>
        <blue>64</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>181</green>
        <blue>64</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>243</green>
        <blue>225</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>243</green>
        <blue>225</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>210 콤퓨타세탁 L</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="dateViewer">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>210 콤퓨타세탁 B</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>date view</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="tagLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>110</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>210 콤퓨타세탁 R</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tag</string>
   </property>
  </widget>
  <widget class="QLabel" name="placeLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>110</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>210 콤퓨타세탁 R</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Place</string>
   </property>
  </widget>
  <widget class="QLabel" name="imageLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>110</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>210 콤퓨타세탁 R</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Food Image</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OKButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>217</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>210 콤퓨타세탁 B</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(143, 86, 59);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="imageView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>241</width>
     <height>121</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Image</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="placeEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="tagEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>170</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>210 콤퓨타세탁 L</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">selection-color: rgb(210, 255, 103);</string>
   </property>
   <item>
    <property name="text">
     <string>Dish</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dessert</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Midnight Snack</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="dishCategory">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>170</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Korean</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Chinese</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Japanese</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Italian</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="star1">
   <property name="geometry">
    <rect>
     <x>305</x>
     <y>100</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>emptystar.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="star2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>100</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>emptystar.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="star3">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>100</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>emptystar.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="star4">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>100</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>emptystar.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="star5">
   <property name="geometry">
    <rect>
     <x>445</x>
     <y>100</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>emptystar.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="rating">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>210 콤퓨타세탁 B</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>RATING</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
